certified-connectors/personr/readme.md

```diff
@@ -0,0 +1,35 @@
+Personr Connector
+
+The Personr connector provides access to a simple, powerful REST API. Using this API, you can create, verify and request the status or more details of an applicant.
```

Comment:
Please update readme.md file as per the template in the below link, please ensure that all the required fields added/updated duly. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md